storage/txpool/Cargo.toml

```diff
@@ -9,30 +9,29 @@ repository  = "https://github.com/Cuprate/cuprate/tree/main/storage/txpool"
 keywords    = ["cuprate", "txpool", "transaction", "pool", "database"]
 
 [features]
-default     = ["heed", "service"]
+default     = ["heed"]
 # default   = ["redb", "service"]
 # default   = ["redb-memory", "service"]
 heed        = ["cuprate-database/heed"]
 redb        = ["cuprate-database/redb"]
 redb-memory = ["cuprate-database/redb-memory"]
-service     = ["dep:tower", "dep:rayon", "dep:cuprate-database-service"]
```

Comment:
Because it was annoying to get this to pass CI so I just removed the feature, almost all use cases will use the tower interface anyway, for those that don't I don't think a few extra dependencies would be too bad.

I for parity removed the feature flag from `cuprate-blockchain` as well and updated the docs